src/content/tutorial/wrapping-up/wrapping-up.mdx

```diff
@@ -25,21 +24,21 @@ Once you have completed all five steps of the tutorial, you can apply for the IB
 
 ### IBM Design System React Components
 
-This badge earner demonstrates knowledge about Carbon's React components. What it takes to earn the badge:
+This badge demonstrates knowledge about Carbon's React components. To earn the badge:
 
-1. Completion of steps 1 through 5 of the React Carbon tutorial
+1. Complete steps 1 through 5 of the React Carbon tutorial
    - Step 1. [Installing Carbon](/tutorial/react-step-1)
    - Step 2. [Building pages](/tutorial/react-step-2)
    - Step 3. [Using APIs](/tutorial/react-step-3)
    - Step 4. [Creating components](/tutorial/react-step-4)
    - Step 5. [Deploying to IBM Cloud](/tutorial/react-step-5)
-2. Submission of _approved_ pull requests for steps 1 through 5 of the React Carbon tutorial in the [carbon-tutorial repository](https://github.com/carbon-design-system/carbon-tutorial).
+2. Submission of approved pull requests for steps 1 through 5 of the React Carbon tutorial in the [carbon-tutorial repository](https://github.com/carbon-design-system/carbon-tutorial).
```

Comment:
Grammar change is not carried through here: 

 "Submission of approved" should now say "Submit approved" - in fact it should say "Submit links to approved pull requests"